.travis.yml

```diff
@@ -0,0 +1 @@
+language: java
```

Comment:
If you want to keep this in, is it ok if I remove it when adding the Jenkinsfile?